test/upgrade/installation/shell.go

```diff
@@ -47,6 +49,7 @@ func callShellFunction(funcName string) error {
 	}
 	exec := shell.NewExecutor(shell.ExecutorConfig{
 		ProjectLocation: loc,
+		Environ:         os.Environ(),
```

Comment:
I don't understand how this is related 🤔

Reply:
This is a bit extra fix that was causing me pain debugging locally.

The upgrade tests use an upgrade framework in `knative.dev/pkg` but it still relies on the bash scripts. So you effectively have an invocation flow of bash => go => bash. 

Not having the environment propagate means things reset. So even though I had my ingress class set it would try to install istio fresh